lib/src/display/render_loop.dart

```diff
@@ -62,7 +74,20 @@ class RenderLoop extends RenderLoopBase {
     }
 
     for (int i = 0; i < _stages.length; i++) {
-      _stages[i].materialize(_currentTime, deltaTime);
+      if(_stages[i].renderMode != StageRenderMode.AUTO_DIRTY || juggler.hasAnimatables || _stages[i].juggler.hasAnimatables || _stages[i].dirty){
```

Comment:
I was thinking about an extension to the Tween class. It already has an callback "onUpdate", you could do something like this today:

    var tween = juggler.addTween(mySprite, 5.0);
    tween.onUpdate = () => stage.RenderMode = StageRenderMode.Once;

It would be possible to introduce a new method:

    tween.autoInvalidateStage(stage);  // TODO: find a better name